modules/nw-multus-tap-object.adoc

```diff
@@ -81,7 +81,66 @@ The following example configures an additional network named `mynet`:
 }
 ----
 
+[id="nw-multus-enable-container_use_devices_{context}"]
+
+== Setting SELinux boolean for the TAP CNI plugin
+
+To create the tap device with the `container_t` SELinux context, enable the `container_use_devices` boolean on the host by using the Machine Config Operator (MCO).
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`).
+
+.Procedure
+
+. Create a new YAML file named for example `setsebool-container-use-devices.yaml` with the following details:
++
+[source, yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: worker
+  name: 99-worker-setsebool
+spec:
+ config:
+ignition:
+version: 3.2.0
+systemd:
+units:
+- contents: |
+    [Unit]
+    Description=Set SELinux booleans
+    Before=kubelet.service
+
+    [Service]
+    Type=oneshot
+    ExecStart=/usr/sbin/setsebool container_use_devices=on
+    RemainAfterExit=true
+
+    [Install]
+    WantedBy=multi-user.target graphical.target
+   enabled: true
+   name: setsebool.service
+----
++
+
+. Create the new `MachineConfig` object by running the following command:
```

Comment:
Please add a note that this change required node reboot and can take few minutes.